.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+<!-- Here is for bug reports and feature requests ONLY! 
+
+If you're looking for help, please check our Dingtalk group and the Gitter room.
```

Comment:
There is no Gitter room now. May replace it by WeChat group.

Reply:
Gitter room is okay here. I'll create a room for it :)